### PR TITLE
Remove user authentication against global and rename EWS in msg

### PR DESCRIPTION
**Pull request by rajivmucheli**
Hi @bbobrov 

Please validate this PR.

Regards,
Rajiv

### CODE REVIEW

File: cc/keystone/auth/plugins/cc_password.py [hunk 1]
```diff
@@ -45,10 +45,10 @@ def __init__(self):
         group = cfg.OptGroup(name='cc_password', title='Outlook Exchange Webservice options')
```
Comment (bbobrov) [inline cc/keystone/auth/plugins/cc_password.py]: this should also be reworded

File: cc/keystone/auth/plugins/cc_password.py [hunk 2]
```diff
@@ -97,17 +97,15 @@ def authenticate(self, auth_payload):
     def _authenticate_ews(username, password):
         """Authenticate a CID user against sap exchange webservice"""
```
Comment (bbobrov) [inline cc/keystone/auth/plugins/cc_password.py]: this description should be changed

File: cc/keystone/auth/plugins/cc_password.py [hunk 3]
```diff
@@ -121,18 +119,18 @@ def _authenticate_ews(username, password):
...
+            LOG.debug("Authenticated %s with Password Verification Service." % username)
             return True
         except exception.Unauthorized:
             raise
         except Exception as e:
             LOG.error(
-                "SAP EWS authentication of '%s' against '%s' failed: %s" % (
+                "SAP Password Verification Service authentication of '%s' against '%s' failed: %s" % (
                     username, CONF.cc_password.url, e))
             raise exception.Unauthorized('Callout to SAP Exchange for password verification failed.')
```
Comment (bbobrov) [inline cc/keystone/auth/plugins/cc_password.py:135]: the description of this exception should also be changed

**Review by bbobrov — CHANGES_REQUESTED** [review]

File: cc/keystone/auth/plugins/cc_password.py (hunk 3 above)
Comment (rajivmucheli) [reply cc/keystone/auth/plugins/cc_password.py:135]: any suggestions ?

Comment (bbobrov) [reply cc/keystone/auth/plugins/cc_password.py:135]: Callout to SAP Password Verification Service failed

Comment (rajivmucheli) [reply cc/keystone/auth/plugins/cc_password.py:135]: sry, i dont get. Please update your msg description if the lastest comment is invalid.

**Pull request merged**